coalib/bearlib/languages/Language.py

```diff
@@ -429,14 +471,105 @@ def limit_versions(language, limit, operator):
         versions = [version for version in language.versions
                     if operator(int(version), limit)]
     else:
-
         versions = [version for version in language.versions
                     if operator(version, limit)]
     if not versions:
         raise ValueError('No versions left')
     return type(language)(*versions)
 
 
+def type_equals(language, comparable):
+    """
+    Checks if the given instances are of the same Language class and raises an
+    error otherwise.
+
+    :param language:
+        A `Language` instance
+    :param comparable:
+        Another `Language` instance
+    :return:
+        True
+    :raises TypeError:
+        If Language instances are not of the same class
+    """
+    if type(language) is type(comparable):
+        return True
```

Comment:
You don't need to `return` anything since either it `raise`s or the code block including it continues